assets/src/app/jobs/jobs.component.html

```diff
@@ -0,0 +1,38 @@
+<h1>
+  Jobs
+</h1>
+
+<ul *ngIf="jobs" class="jobs">
+  <li *ngIf="jobs.total == 0">Aucun résultat trouvé</li>
+  <li
+    *ngFor="let job of jobs.data"
+    class="job {{job.channel}} {{job.status | jobStatus}}"
+    >
+    <span *ngIf="job.name == 'ftp_order'" class="icon">
+      <mat-icon>compare_arrows</mat-icon>
+    </span>
+    <span class="title">
+      {{job.name | jobType}}
+    </span>
+    <span class="status">
+      {{job.status | jobStatus}}
+    </span>
+    <span class="date">
+      {{job.inserted_at | date: 'd LLL yy'}}
```

Comment:
We could create a specific PipeTransform that would use Moment.js to handle date internationalization, and so display it in French in this case, don't you think?

Reply:
A little overkill at this step I think.